gbpservice/nfp/bin/nfp_configurator.ini

```diff
@@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
 reportstate_interval=10
 periodic_interval=9
 
+log_forward_ip_address=
+log_forward_port=514
```

Comment:
If we change this port, will things work on the configured (other than 514) port?